circle.yml

```diff
@@ -642,6 +643,23 @@ commands:
           fi
 
 jobs:
+  next-android-api-breakage:
+    executor: ubuntu-disco
+    steps:
+      - run:
+          name: Clone mapbox-gl-native-android in working directory
+          command: git clone https://github.com/mapbox/mapbox-gl-native-android.git .
+      - next-prepare
+      - run:
+          name: Init submodules
+          command: git submodule update --init --recursive
+      - run:
+          name: Update mapbox-gl-native to match PR
+          command: cd vendor/mapbox-gl-native && git fetch && git checkout $CIRCLE_SHA1 && cd .. && git submodule update --init --recursive
```

Comment:
This will checkout gl-native back to the original HASH. The `cd ..` has to be after the `submodule update`.

Reply:
And I don't think the `git fetch` is needed.